Makefile

```diff
@@ -39,9 +39,9 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # This variable is used to construct full image tags for bundle and catalog images.
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# cassandra.datastax.com/cass-oper-bundle:$VERSION and cassandra.datastax.com/cass-oper-catalog:$VERSION.
+# k8ssandra/cass-operator-bundle:$VERSION and k8ssandra/cass-operator-catalog:$VERSION.
 
-ORG ?= k8ssandra
+ORG ?= us-docker.pkg.dev/k8ssandra/images
```

Comment:
This will break all local builds.

Reply:
Why? It is working for me locally. The presence of a domain only affects pulling and pushing. I can import this package into Kind without issue.

Reply:
It's only working for certain use-cases. See, now you're overriding the default repository - not just the org. If I want to use a localhost:5000 repository (local registry) - which I've setup for example, this won't work. The images now become: localhost:5000/us-docker.pkg.dev/k8ssandra/images - and this is invalid. If you set the structure to be:

Same applies to the image_config.yaml. If you wish to override the default repository, use the ``imageRegistry`` field, otherwise everyone who uses their own local registry (not just localhost, but say a proxy also) will not be able to use that feature anymore. Our parsing does not understand that there's repository applied in the wrong field - it assumes the format for certain image structures is ``organization/image`` and the additional repository comes from a different property.

Reply:
Alright, I'm understanding the relationships in `image_config.yaml` but there's a bunch of overlap. Why are we baking in the `ORG` as well, shouldn't that be part of the registry definition?

So if I want to use `docker.io` why not just specify `imageRegistry: docker.io/k8ssandra` with the image name `system-logger:latest`?

There's another gotcha in that we don't host the DSE images within the `k8ssandra` org, there isn't a way for me to specify the `imageRegistry` for particular images and not others. Should we mirror the the images with management API to our org?

Reply:
For the `Makefile`, should we add a `REGISTRY` parameter that can be overridden?